sync/class.jetpack-sync-module-plugins.php

```diff
@@ -54,6 +56,61 @@ public function check_upgrader( $upgrader, $details) {
 		}
 	}
 
+
+	public function check_plugin_edit() {
+		$screen = get_current_screen();
+		if ( 'plugin-editor' !== $screen->base ||
+			! isset( $_POST['action'] ) ||
```

Comment:
We don't need to check for action == update. Since `admin_action_update` only get fired when that is the case. 

Lets check that for `$_POST['newcontent']`